services/web-server/.neutrinorc.js

```diff
@@ -0,0 +1,36 @@
+const nodeExternals = require('webpack-node-externals');
```

Comment:
There seems to be 2 `.neutrinorc.js` files. Let's delete `.neutrinorc.js` and keep `services/web-server/.neutrinorc.js`.

Reply:
yeah, a left-over from Friday banging on the wall :)